core/getting_started.html

```diff
@@ -26,6 +26,5 @@
 
   </div>
 </div>
-<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
Agreed, missed that, but let's then change the jQuery version to be 2.x instead of 1.x that is in the head include.